src/lib.rs

```diff
@@ -98,6 +100,17 @@ pub struct TimerHeap<T> {
     counter: u64
 }
 
+impl<T:Debug + Eq + Clone + Hash + Ord> Debug for TimerHeap<T> {
+    fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
+        fmt.debug_map()
+           .entries(self.timers
+                        .iter()
+                        .filter(|e| self.is_active(e))
+                        .map(|e| (e.key.clone(), DebugEntry::from(e))))
```

Comment:
I don't think you should need to clone the key here given that `&T: Debug where T: Debug`.

Reply:
I did this because I was getting a borrowck error about moving out of e. But maybe I can just replace this with &e.key. I'll try that when I get home from work.

Thanks!

Reply:
Yeah, I think `&e.key` should work.

Reply:
Yep. That worked. I'll merge it in, and cut a release.